src/components/NavBar/LightDarkSlider.tsx

```diff
@@ -0,0 +1,45 @@
+import { Switch } from "@mui/material";
+import styled from "styled-components";
+
+const LightDarkSlider = styled(Switch)`
+    width: 62px;
+    height: 34px;
+    padding: 7px;
+    & .MuiSwitch-switchBase {
+        margin: 1px;
+        padding: 0;
+        transform: translateX(6px);
+        &.Mui-checked {
+            transform: translateX(22px);
+            & .MuiSwitch-thumb:before {
```

Comment:
does this have to be nested? how is this different to the below `.MuiSwitch-thumb` (apart from being `:before`)?

Reply:
It does seem to be, have tried moving bits about and this is the cleanest I've managed to get it.